apps/src/applab/designModeHeaders.jsx

```diff
@@ -0,0 +1,17 @@
+var React = require('react')
+var applabMsg = require('./locale')
+
+module.exports = React.createClass({
+  render: function() {
+    return (
+      <div>
+        <div id="design-toolbox-header" className="workspace-header">
+          <span>{applabMsg.designToolboxHeader()}</span>
+        </div>
+        <div id="design-workspace-header" className="workspace-header">
+          <span>{applabMsg.designWorkspaceHeader()}</span>
+        </div>
+      </div>
+    )
+  }
+});
```

Comment:
For this and the previous file. When writing a .jsx file that exports a class, I've been trying to use upper cases leading letters (DesignModeHeaders instead of designModeHeaders).